vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -1042,18 +1054,56 @@ ${space.description}
 <button type="button" id="addModuleLinkButton" class="btn btn-primary btn-sm">Add Module Link</button> &nbsp
 <button type="button" id="addExternalLinkButton" class="btn btn-primary btn-sm">Add External Link</button> &nbsp
 <button type="button" id="changeBgImgButton" class="btn btn-primary btn-sm"> Change Image</button> &nbsp
-<button type="button" class="btn btn-primary btn-sm" data-record-id="${space.id}" data-url="<c:url value="/staff/space/${space.id}?${_csrf.parameterName}=${_csrf.token}"/>" data-call-on-success = "<c:url value="/staff/space/list"/>" data-call-on-error="<c:url value="/staff/space/list"/>" data-toggle="modal" data-target="#confirm-delete">
-Delete Space
-</button>
+<button type="button" id="deleteSpace" class="btn btn-primary btn-sm">Delete Space</button>
+
 </nav>
 
 <p></p>
 
+<div class="modal fade" id="confirm-space-delete" tabindex="-1" role="dialog"
+    aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="deleteModalTitle">
+                    Confirm ${space.id} Deletion?
+                </h5>
+                <button type="button" class="close" data-dismiss="modal"
+                    aria-hidden="true">×</button>
+            </div>
+            <div class="modal-body">
+                <p>
+                    Are you sure you want to delete ${space.id}?
+                </p>
+                <div id="warningMessage">
+                <small class="text-danger">
+                Other spaces have links to this space!
+                </small>
+                </div>
+                <div id="exhibitionMessage">
+                <small class="text-danger">
+                This space is the exhibition start space! This will set exhibition to null.
```

Comment:
Should say: "This space is the start of the exhibition. Deleting it will make your exhibition unavailable."